### PR TITLE
Remove angular

### PR DESCRIPTION
**Pull request by adamgrossman**

### CODE REVIEW

File: feed/static/js/controllers/linkController.js [hunk 1]
```diff
@@ -1,5 +0,0 @@
-function linkController($scope, GroupFactory, LinkFactory) {
-    var linkID = $routeParams.id;
-
-    console.log(linkID);
```
Comment (gkadillak) [inline feed/static/js/controllers/linkController.js]: brilliant work

File: feed/static/js/factories/linksFactory.js [hunk 2]
```diff
@@ -1,45 +0,0 @@
-peer_post.factory('LinkFactory', function ($http) {
-    return {
-        linksList: [],
```
Comment (gkadillak) [inline feed/static/js/factories/linksFactory.js]: good call

File: feed/static/js/vendors/angular/angular-route.js [hunk 3]
```diff
@@ -1,982 +0,0 @@
...
-            }
-          }).
-          // after route change
-          then(function(locals) {
-            if (nextRoute == $route.current) {
-              if (nextRoute) {
-                nextRoute.locals = locals;
-                angular.copy(nextRoute.params, $routeParams);
-              }
-              $rootScope.$broadcast('$routeChangeSuccess', nextRoute, lastRoute);
```
Comment (gkadillak) [inline feed/static/js/vendors/angular/angular-route.js]: ah I see what you did there

File: feed/static/js/views/home.html [hunk 4]
```diff
@@ -1,80 +0,0 @@
...
-                    <button type="submit" size="xs" ng-click="newLink()">Submit</button>
-                </div>
-            </div>
-        </div>
-
-        <div ng-repeat="link in allLinks">
-            <div class="link-info" ng-class-odd="'odd'" ng-class-even="'even'">
-                <div class="row">
-                    <div class="col-xs-1">
-                        <div class="score-div">
```
Comment (gkadillak) [inline feed/static/js/views/home.html]: semantics could be a little polished up here fyi.

**Comment by gkadillak** [comment]
:+1:  after comments of course

File: feed/static/js/views/home.html (hunk 4 above)
Comment (adamgrossman) [inline feed/static/js/views/home.html]: you're right. this felt "a bit lazy"

**Pull request merged**